Sequence.cpp

```diff
@@ -2,96 +2,136 @@
  *  RemoteArDuino (RAD) Gate
  *  (c) Danny Frencham 2017
  *****************************************/
+ /**************************************************
+  Action Plan for the latter part of sequence
+
+ ItemTime(ms)Action1  Action2          Action3
+  0    0     Set LED1  Play Tone 632 
+  1   60               Stop Tone 
+  2  120     Set LED2  Play Tone 632 
+  3  180               Stop Tone 
+  4  240     Set LED3  Play Tone 632 
+  5  300               Stop Tone 
+  6  360     Set LED4  Play Tone 632   Gate Activate
+  7  660                               Gate Deactivate
```

Comment:
I'm not sure why we need an explicit "Deactivate" state. After the gate drops it is deactivated (magnet off, no sequence running). I can understand an "armed" state, as that implies the magnet is powered.

Reply:
The solenoid just requires a pulse.  300 ms seems long enough. Leaving it on sucks a lot of power and will surely  burn out the coil. I tried making a pulse by dumping charge from 50000uF worth of charge from some capacitors - not enough. So software is better, and  the need to deactivate. The The Gate::arm() function was empty so used it... should have renamed it

Reply:
ok, cheers makes sense. Should we have a variable name gateControlType that we can set to "solenoid" (with default being "magnet")?

That way we can change the behaviour based on that setting.